deployment/local-dev/pgadmin-deployment.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pgadmin-auth
+  namespace: languagedepot
+stringData:
+  PGADMIN_DEFAULT_EMAIL: 'admin@test.com'
+  PGADMIN_DEFAULT_PASSWORD: 'pass'
```

Comment:
let's just inline these as environment variables since this is local only

Reply:
Done in commit 47585fdd.